src/core/lib/security/credentials/tls/grpc_tls_certificate_provider.cc

```diff
@@ -413,6 +486,19 @@ absl::StatusOr<bool> PrivateKeyAndCertificateMatch(
   return result;
 }
 
+absl::Status IsKeyCertPairsListValid(const PemKeyCertPairList& pair_list) {
+  if (!pair_list.empty()) {
```

Comment:
This check is not required. For loop won't execute if list is empty.

Reply:
Makes sense, thanks!